frontend/src/components/general/menu/hamburger/hamburger.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  const dispatch = createEventDispatcher();
+  const click = () => dispatch('click');
+</script>
+<button on:click aria-label="Menu">
+	<svg id="icon" width="184" height="184" viewBox="0 0 184 184" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
why do this rather than just extract the button and use svelte icon thing?

Reply:
cuz I copied it from stackoverflow and that is more effort. But that is a fair point. TODO later?